code/hsec-tools/src/Security/Advisories/Git.hs

```diff
@@ -117,4 +119,7 @@ getAdvisoryGitInfo path = do
       -- the same as `ExitFailure`
       pure . Left $ GitProcessError status stdout stderr
   where
-    parseTime s = maybe (Left $ GitTimeParseError s) Right $ iso8601ParseM s
+    parseTime :: String -> Either GitError ZonedTime
+    parseTime s = maybe (Left $ GitTimeParseError s) Right $
+       iso8601ParseM s
+         <|> utcToZonedTime utc <$> iso8601ParseM s
```

Comment:
I'm not sure it does preserve the date

Reply:
I tried to pass `--date=local` and it didn't work. I'm starting to think that a non-zoned time should be a parse into `UTCTime`.

Reply:
> I'm not sure it does preserve the date

what do you mean?

Reply:
I just validated that the osv files are not changed by this PR with:

```
mkdir osvs
for i in $(find advisories/ -name "*.md" | grep -v reserved); do cabal run exe:hsec-tools -- osv $i > osvs/$(basename $i); done
```

Reply:
It's not that obvious, we get a time and day without time zone, assuming UTC seems audacious. 

IMO, we should rely only on UTCTime, enforcing it in the git command.

Reply:
Oh I see, yes I'm on UTC. Then I remove my approval. Perhaps we could just parse an epoch timestamp if possible?

Reply:
hsec-tools already does this several times (assuming UTC when there's not enough information) That's why I opened the issue about removing ZonedTime from hsec-tools. It makes stuff harder as well. It *is* however the correct *time* to assume, it's not *wrong*, it's just not semantically as nice. See https://github.com/haskell/security-advisories/issues/203 cc @frasertweedale who commented on #203